app/controllers/project.py

```diff
@@ -264,9 +267,31 @@ def get(self):
                 base_query = base_query.filter(getattr(Project, key) == value)
 
 
+        pending_invitations =  []
+
         if not has_role(current_user_roles, 'administrator'):
-            base_query = base_query.filter(or_(Project.owner_id == current_user_id, Project.users.any(
-                ProjectUser.user_id == current_user_id)))
+            base_query = base_query.filter(or_(Project.owner_id == current_user_id,
+                Project.users.any(
+                    and_(
+                        ProjectUser.user_id == current_user_id,
+                        ProjectUser.accepted_collaboration_invite == True
+                    )
+                )
+            ))
+            pending_invitations_query = Project.query.filter(
+                Project.owner_id != current_user_id, 
+                Project.users.any(
+                    and_(
+                        ProjectUser.user_id == current_user_id,
+                        ProjectUser.accepted_collaboration_invite == False
+                    )
+                )
+            ).all()
+
+            pending_projects_data, pending_projects_errors = project_schema.dumps(pending_invitations_query)
+            
+            if pending_projects_errors:
```

Comment:
This will cause a bug, should be:

```suggestion
            if not pending_projects_errors:
```

Reply:
thanks, sorry for that

Reply:
nice catch sir sorry for this